docs/resources/epg_useg_ip_attribute.md

```diff
@@ -0,0 +1,133 @@
+
+* Class: [fvIpAttr](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvIpAttr/overview)
+
+* Supported in ACI versions: 1.1(1j) and later.
+
+* Distinguished Name Format: `uni/tn-{name}/ap-{name}/epg-{name}/crtrn/ipattr-{name}`
+
+## GUI Information ##
+
+* Location: `Tenants -> Application Profiles -> uSeg EPGs -> uSeg Attributes`
+
+## Example Usage ##
+
+The configuration snippet below creates a EPG uSeg IP Attribute with only required attributes.
+
+```hcl
+
+resource "aci_epg_useg_ip_attribute" "example_epg_useg_block_statement" {
+  parent_dn = aci_epg_useg_block_statement.example.id
+  ip        = "131.107.1.200"
+  name      = "131"
+}
+
+```
+The configuration snippet below shows all possible attributes of the EPG uSeg IP Attribute.
+
+!> This example might not be valid configuration and is only used to show all possible attributes.
+
+```hcl
+
+resource "aci_epg_useg_ip_attribute" "full_example_epg_useg_block_statement" {
+  parent_dn     = aci_epg_useg_block_statement.example.id
+  annotation    = "annotation"
+  description   = "description"
+  ip            = "131.107.1.200"
+  name          = "131"
+  name_alias    = "name_alias"
+  owner_key     = "owner_key"
+  owner_tag     = "owner_tag"
+  use_fv_subnet = "yes"
+  annotations = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+  tags = [
+    {
+      key   = "key_0"
+      value = "value_1"
+    }
+  ]
+}
+
+```
+
+All examples for the EPG uSeg IP Attribute resource can be found in the [examples](https://github.com/CiscoDevNet/terraform-provider-aci/tree/master/examples/resources/aci_epg_useg_ip_attribute) folder.
+
+## Schema ##
+
+### Required ###
+
+* `parent_dn` - (string) The distinguished name (DN) of the parent object, possible resources:
+  - [aci_epg_useg_block_statement](https://registry.terraform.io/providers/CiscoDevNet/aci/latest/docs/resources/epg_useg_block_statement) ([fvCrtrn](https://pubhub.devnetcloud.com/media/model-doc-latest/docs/app/index.html#/objects/fvCrtrn/overview))
+* `ip` (ip) - (string) The device IP address of the EPG uSeg IP Attribute object.
+* `name` (name) - (string) The name of the EPG uSeg IP Attribute object.
+
+### Read-Only ###
+
+* `id` - (string) The distinguished name (DN) of the EPG uSeg IP Attribute object.
+
+### Optional ###
+  
+* `annotation` (annotation) - (string) The annotation of the EPG uSeg IP Attribute object.
+  - Default: `orchestrator:terraform`
+* `description` (descr) - (string) The description of the EPG uSeg IP Attribute object.
+* `name_alias` (nameAlias) - (string) The name alias of the EPG uSeg IP Attribute object.
+* `owner_key` (ownerKey) - (string) The key for enabling clients to own their data for entity correlation.
+* `owner_tag` (ownerTag) - (string) A tag for enabling clients to add their own data. For example, to indicate who created this object.
+* `use_fv_subnet` (usefvSubnet) - (string) The usefvSubnet flag of the EPG uSeg IP Attribute object.
```

Comment:
what does this do, and should we rename it to `use_subet` or something else? I would personally avoid class naming in the attributes names

Reply:
UI says use_epg_subnet